docker/base/Dockerfile

```diff
@@ -15,6 +15,36 @@
 # NOTE: We try to follow the guide in https://docs.docker.com/develop/develop-images/dockerfile_best-practices/
 #       Each RUN contains a bundle of steps, which reduces the cache.
 
+from golang:1.19 AS go_builder
+
+RUN set -eux; \
```

Comment:
Should we specify some tags or commits for those repositories?

Reply:
Yes, we should.